js/src/ui/TxList/txList.js

```diff
@@ -35,19 +35,16 @@ class TxList extends Component {
       PropTypes.array,
       PropTypes.object
     ]).isRequired,
-    netVersion: PropTypes.string.isRequired
+    blockNumber: PropTypes.object,
+    netVersion: PropTypes.string.isRequired,
+    newError: PropTypes.func
   };
 
-  store = new Store(this.context.api);
-
   componentWillMount () {
+    this.store = new Store(this.context.api, this.props.newError);
```

Comment:
newError is a redux action. So I don't really know what you mean. Basically it goes like this:

action->reducer->store->component.

passing in an argument to newError is the action phase.